style.css

```diff
@@ -1,3 +1,6 @@
-h1 {
-  color: blueviolet;
-}
+#title {
+  overflow: hidden;
+  background-color: #f1f1f1;
+  padding: 20px 10px;
+  text-align: center;
+}
```

Comment:
please check your style lint you can check that by installing the linters in your workstation, here is a very useful information [https://github.com/microverseinc/linters-config/tree/master/html-css](url)